test/integration/consul-container/test/consul_envoy_version/consul_envoy_version.go

```diff
@@ -34,7 +34,7 @@ func main() {
 	sort.Sort(sort.Reverse(sort.StringSlice(cev.EnvoyVersions)))
 
 	ceVersions := consulEnvoyVersions{
-		ConsulVersion: string(cVersion),
+		ConsulVersion: cVersion,
```

Comment:
This was causing issues with the linter.

Reply:
It looks like this change has [already been applied to `main`](https://github.com/hashicorp/consul/blob/main/test/integration/consul-container/test/consul_envoy_version/consul_envoy_version.go#L37). You may need to merge/rebase.